scripts/generate_assets.go

```diff
@@ -16,7 +16,10 @@ func main() {
 		log.Fatal("Require 2 args")
 	}
 	directory := os.Args[1]
-	buildTag := os.Args[2]
+	buildTag := ""
+	if len(os.Args) > 2 {
+		buildTag = os.Args[2]
+	}
```

Comment:
When `$BUILD_TAG_PARAMETER` is empty, `len(os.Args) == 2`, and `os.Args[2]` would crash.

Reply:
Could you try with `"$BUILD_TAG_PARAMETER"` instead of `$BUILD_TAG_PARAMETER`?

Reply:
Yep, that's would be better, let me try it.